docs/develop/dotnet/failure-detection.mdx

```diff
@@ -125,6 +125,21 @@ return await Workflow.ExecuteActivityAsync(
     });
 ```
 
+### Overriding the retry interval with Next Retry Delay {#next-retry-delay}
```

Comment:
```suggestion
### Override the retry interval with `NextRetryDelay` {#next-retry-delay}
```

- We use infinitive/base form, not present participle.
- When using specific code-derived terms, such as field names, use code face without spaces.

Reply:
This is what is already present at `docs/develop/java/failure-detection.mdx` and may be a rule worth applying consistently there too instead of just here (but feel free to apply here first)